### PR TITLE
Added TarunKumar in the Mentor list

### PR DESCRIPTION
**Pull request by meets2tarun**
Added my info in the mentor list as asked for the FOSSASIA Advanced Developer Program 2017

### CODE REVIEW

**Comment by mariobehling** [comment]
Please resolve conflicts.

**Comment by isuruAb** [comment]
@meets2tarun  could you please resolve these conflicts

File: index.html [hunk 1]
```diff
@@ -956,6 +956,18 @@ <h1 class="text-white">About "In the Heat of the Code"</h1>
 						<div class="col-md-3 col-sm-6 speaker-column">
 							<div class="speaker">
 								<div class="image-holder">
+									<img class="background-image" alt="Tarun Kumar" src="img/TarunKumar.jpg">
```
Comment (isuruAb) [inline index.html]: there are details of two mentors in one column please check it agian

Comment (meets2tarun) [reply index.html]: I can't find that. Everything working fine locally!

Comment (niccokunzmann) [reply index.html]: @isuruAb Can you provide a screenshot or a correction?

Comment (isuruAb) [reply index.html]: @niccokunzmann  send a pr to @meets2tarun  fork. 👍

**Comment by meets2tarun** [comment]
Thank you @isuruAb  👍

**Pull request merged**